apps/labs/posts/unlocking-c-level-performance-in-df-apply.md

```diff
@@ -0,0 +1,466 @@
+---
+title: 'Unlocking C-level performance in pandas DataFrame.apply with numba'
```

Comment:
Nice title, I like it. Minor: perhaps add a `:` (or a `'`) after `pandas` to connect the two parts of the title? Also, Numba with a capital N (also on line 5).

Reply:
Changed it to pandas.DataFrame.apply, that should sound smoother.